scg-system/src/test/java/io/telicent/TestInitialCompaction.java

```diff
@@ -84,12 +84,13 @@ public void test_authMemoryDataset() {
     public void test_persistentDataset() {
         // given
         mockDatabaseMgr.when(() -> DatabaseMgr.compact(any(), anyBoolean())).thenAnswer(invocationOnMock -> null);
+        mockDatabaseMgr.clearInvocations();
```

Comment:
Double counting mystery solved?

Reply:
Yup. Bug in the underlying static mocking code. 

The trouble is to do with trying to mock the static void method. I had all manner of other trouble with the Dataset tests and so took a different approach. I couldn't (and still don't) understand why it was working for the compaction but not the back-up tests but thought it worth correcting here.